helm/sematic-server/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sematic-server
 description: Sematic AI Server
 type: application
-version: 1.1.7
+version: 1.1.8
```

Comment:
this should only be done when we actually cut the next release, otherwise it'll get double-incremented